bitsrun/config.py

```diff
@@ -17,13 +17,22 @@ def get_config_paths() -> map:
     - `C:\ProgramData\bitsrun\bit-user.json`
     - `~\AppData\Roaming\bitsrun\bit-user.json`
 
-    On macOS and Linux:
+    On Linux:
 
     - `/etc/bitsrun/bit-user.json`
+    - `/etc/xdg/bitsrun/bit-user.json`
     - `$XDG_CONFIG_HOME/bitsrun/bit-user.json`
     - `~/.config/bitsrun/bit-user.json`
     - `~/.config/bit-user.json`
```

Comment:
4-line version might be less confusing …

```bash
# Test on Debian

$ bitsrun config-paths
/etc/bit-user.json
/etc/xdg/bitsrun/bit-user.json
/home/<user>/.config/bitsrun/bit-user.json
/home/<user>/.config/bit-user.json

$ XDG_CONFIG_DIRS=/whatever bitsrun config-paths
/etc/bit-user.json
/whatever/bitsrun/bit-user.json
/home/<user>/.config/bitsrun/bit-user.json
/home/<user>/.config/bit-user.json

$ XDG_CONFIG_HOME=/this-is-home bitsrun config-paths
/etc/bit-user.json
/etc/xdg/bitsrun/bit-user.json
/this-is-home/bitsrun/bit-user.json
/this-is-home/bit-user.json
```

IMHO developers should check `platformdirs`' doc / source codes for explanation, not ours.

However, (if it really counts) I suggest the following.

- `/etc/bitsrun/bit-user.json`
- `$XDG_CONFIG_DIRS/bitsrun/bit-user.json` (exclusive) or `/etc/xdg/bitsrun/bit-user.json`
- `$XDG_CONFIG_HOME/bitsrun/bit-user.json` (exclusive) or `~/.config/bitsrun/bit-user.json`
- `$XDG_CONFIG_HOME/bit-user.json` (exclusive) or `~/.config/bit-user.json`